lib/websocket_server.rb

```diff
@@ -24,7 +26,10 @@ def initialize
         reads.each do |read|
           begin
             @pairing[read].proxy.transmit(writes, @pairing[read].is_ws)
-          rescue
+          rescue IOError
+            cleanup(errors)
+          rescue => ex
+            @logger.error("#{ex.class} for #{@pairing[read].proxy.vm_id}: #{ex.message}\n#{ex.backtrace.join("\n")}")
```

Comment:
Perhaps we want to move this error handling/message building `@pairing[read].proxy.vm_id` down into the proxy so you don't have do it here.